neuroquery/datasets.py

```diff
@@ -72,12 +73,19 @@ def fetch_peak_coordinates(data_dir=None):
         return str(out_file)
     print("Downloading coordinates")
     resp = requests.get(
-        "https://raw.githubusercontent.com/neuroquery/neuroquery_data/"
-        "master/training_data/coordinates.csv"
+        "https://raw.githubusercontent.com/neuroquery/neuroquery_data/master/data/data-neuroquery_version-1_coordinates.tsv.gz"
     )
     resp.raise_for_status()
+    content = zlib.decompress(resp.content, 15 + 32)
+    content_lines = content.split(b"\n")
```

Comment:
as neuroquery depends on pandas already, I think we can use it to parse and write the csv. the code will be a bit simpler, and it will produce correct results if for some weird reason some table id or table name contains a tab or a newline.
to do so we can (i) create a `io.BytesIO` buffer that offers a file object -like interface (ie a `read()` method) to the bytes in `content`:
```python
    content_buf = io.BytesIO(content)
```
(ii) parse the csv using pandas `read_csv` and (iii) write the csv with `DataFrame.to_csv`:

```python
    df = pd.read_csv(content_buf, sep="\t")
    df.to_csv(str(out_file), index=False)
```

so in the end the final part of the function might look like
```python
    content = zlib.decompress(resp.content, 15 + 32)
    content_buf = io.BytesIO(content)
    df = pd.read_csv(content_buf, sep="\t")
    df.to_csv(str(out_file), index=False)
    print("Downloaded.")
    return out_file
```

does that make sense?